static/app/components/charts/worldMapChart.tsx

```diff
@@ -59,7 +59,8 @@ class WorldMapChart extends React.Component<Props, State> {
       import('sentry/data/world.json'),
     ]);
 
-    echarts.registerMap('sentryWorld', worldMap.default as any);
+    // Echarts not available in tests
+    echarts.registerMap?.('sentryWorld', worldMap.default as any);
```

Comment:
🤢

Reply:
i played around with module.exports for a while but couldn't figure out how to get it to return the right thing